README.md

```diff
@@ -235,6 +235,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker/)
 
 ### Orchestration
 
+-   [Ansible Linux Docker](https://github.com/Peco602/ansible-linux-docker) - Run Ansible from a Linux container. By [Peco602](https://github.com/Peco602)
```

Comment:
https://github.com/devture/docker-ansible/blob/master/Dockerfile looks more minimalistic and is based on Alpine Linux, which brings image size down by a vast amount.
You mentioned that you're "not sure the latter supports also Windows hosts. Looking at Dockerfile it seems not.". Sadly i don't have a Windows computer available to test your claim, but for me it looks fine. What were the reasons that you came to this comclusion?

Reply:
https://github.com/devture/docker-ansible/blob/master/Dockerfile performs just a standard Ansible installation. https://github.com/Peco602/ansible-linux-docker/blob/main/Dockerfile instead is much more complete since it also installs packages like `pywinrm` and `pykerberos` which allow the usage of Ansible on Windows targets and add the Kerberos support for authentication in AD environment. As you can see, these packages are absent on the first Dockerfile.

Reply:
@Peco602 thank you for the explanation, sounds logical to me to also have these packages included, especially to spare new users from having to deep dive into a potential extensive setup and customisation process.